api/TimeAddressableMediaStore.yaml

```diff
@@ -924,7 +1128,17 @@ paths:
           description: Bad request. Invalid query options.
     put:
       summary: Create or Replace Flow
-      description: Create or replace the flow metadata.
+      description: |
+        Create or replace the flow metadata.
+
+        Flows can be created either by directly supplying all the required technical metadata or by referencing
+        an existing profile in the store.  When using a profile the store will combine the supplied flow metadata
+        with the profile metadata such that on reading the flow a complete set of flow metadata is supplied.  It
```

Comment:
"When using a profile" can unintentionally be read as a way to override the profile settings (because it's not clear that "flow metadata" and "profile metadata" don't overlap). The restriction on doing so is explained in the second paragraph, but does this better convey that message here? Or maybe the notion that trying to modify a profile at Flow creation is an error should be moved up a bit?

> When using a profile, the store will use the technical details (`format`, `codec`, `essence_parameters` etc.) from that profile, along with the metadata provided.